### ISSUE TITLE
[UI Improvement] <Would like to improve the UI of website>

### ISSUE THREAD

**Issue by imsp18** [KiranAminPanjwani/MedStats#16]
### 🎇 Description

Want to Improve the UI of website to make it more user-friendly and easy to use.

### 🖼 Screenshots

Proposed Changes:

![image](https://github.com/KiranAminPanjwani/MedStats/assets/108970936/dda7688b-2efa-48e6-a2fd-1cdbc6035c07)

Link to Figma File:
https://www.figma.com/file/DyKgrkyXBnUBuZVbVrwQ9M/Hacktoberfest---MedStats?type=design&node-id=0%3A1&mode=design&t=OqCGbA1tpuuG8W8Q-1

**Comment by KiranAminPanjwani** [comment]
That's fantastic! 😇✨ Please update your forked repository and proceed.

However, ensure that the design allows users to differentiate between the red, green, yellow, and range colors on the progress bar. I suggest using a light theme with a white background. Also, remember to include the Medstats logo at the top.

**Comment by imsp18** [comment]
Yes. Ok will make sure to make color visible, keep the theme light and use the orginal logo.

**Comment by KiranAminPanjwani** [comment]
@imsp18 Updates?

**Comment by Albert-2** [comment]
I'll love to contribute can you please assign me this issue
I'll try my best..
THANK YOU..!!:)

**Comment by imsp18** [comment]
@KiranAminPanjwani working on it, will my try my best to do it in 2 days.

**Comment by KiranAminPanjwani** [comment]
> I'll love to contribute can you please assign me this issue I'll try my best.. THANK YOU..!!:)

There is room for so many improvements on the website. Kindly visit the website again and navigate it. I'm sure you will find many bugs :-)
Else visit #66
